Assets/MRTK/Core/Services/BaseEventSystem.cs

```diff
@@ -128,8 +128,9 @@ public override string ToString()
 
             if (eventExecutionDepth == 0 && (postponedActions.Count > 0 || postponedObjectActions.Count > 0))
             {
-                foreach (var handler in postponedActions)
+                for (var i = 0; i < postponedActions.Count; i++)
                 {
+                    var handler = postponedActions[i];
```

Comment:
I know this used `var` before, but since you're making a change here, could you also remove the usage of `var` so we actually know what type this is? It's above in the `for` loop declaration as well!

Reply:
Updated this!